pkg/controllers/machinesync/machine_sync_controller.go

```diff
@@ -539,13 +544,13 @@ func (r *MachineSyncReconciler) reconcileMAPIMachinetoCAPIMachine(ctx context.Co
 		BlockOwnerDeletion: ptr.To(true),
 	}})
 
-	result, syncronizationIsProgressing, err := r.createOrUpdateCAPIInfraMachine(ctx, mapiMachine, infraMachine, newCAPIInfraMachine)
+	result, synchronizationIsProgressing, err := r.createOrUpdateCAPIInfraMachine(ctx, mapiMachine, infraMachine, newCAPIInfraMachine)
 	if err != nil {
 		return result, fmt.Errorf("unable to ensure Cluster API infra machine: %w", err)
 	}
 
-	if syncronizationIsProgressing {
-		return ctrl.Result{RequeueAfter: time.Second * 1}, r.applySynchronizedConditionWithPatch(ctx, mapiMachine, corev1.ConditionUnknown,
+	if synchronizationIsProgressing {
+		return ctrl.Result{RequeueAfter: requeueInfraProgress}, r.applySynchronizedConditionWithPatch(ctx, mapiMachine, corev1.ConditionUnknown,
```

Comment:
Is there any object whose state affects the value of `synchronizationIsProgressing` which this controller doesn't watch? If not we shouldn't requeue here.

Also, this pattern makes C-R grumpy if `applySynchronizedConditionWithPatch` because we've returned an error and a non-empty Result. It moans about it. We should split this out and return the error separately.

Reply:
This is just changing our existing requeue to use a const, let's tackle this in the follow up that updates the watch filters. Will remove the change from this PR.